openshift/oadp-operator
# OADP-2747: Re-enable DataMover E2E tests

Update DataMover E2E tests to run against Velero builtin DataMover.

## How to test

Read Testing documentation and run `make test-e2e`. Check if DataMover was invoked (`DataUploads` and `DataDownloads` instances were created).
